Makefile

```diff
@@ -68,10 +74,12 @@ publish-docker-image: build-docker-image
 	docker push $(DOCKER_REPO_NAME)/ralph:latest
 	docker push $(DOCKER_REPO_NAME)/ralph-static-nginx:$(RALPH_VERSION)
 	docker push $(DOCKER_REPO_NAME)/ralph-static-nginx:latest
+	docker push $(DOCKER_REPO_NAME)/inkpy:$(RALPH_VERSION)
```

Comment:
For now I didn't want to push with tag "latest". I'm waiting for response from Kamil Warguła how we should proceed with inkpy updates